content-security-policy/support/testharness-helper.sub.js

```diff
@@ -0,0 +1,47 @@
+const Host = {
+  SAME_ORIGIN: "same-origin",
+  CROSS_ORIGIN: "cross-origin",
+};
+
+const PolicyHeader = {
+  CSP: "echo-policy.py?policy=",
+  EMBEDDING_CSP: "echo-embedding-csp.py",
+} 
+
+function generateURL(host, path) {
+  var url = new URL("http://{{host}}:{{ports[http][0]}}/content-security-policy/support/");
```

Comment:
Ahhh.. Thinking that Embedding-CSP should be sent no matter what, I forgot that however post message wouldn't be sent if it is not loaded! All tests fixed ( except the one on the wrong value of csp that we don't validate)